pkg/apis/planetscale/v2/etcdlockserver_types.go

```diff
@@ -126,6 +126,42 @@ type EtcdLockserverTemplate struct {
 	// ExtraLabels can optionally be used to attach custom labels to Pods
 	// created for this component.
 	ExtraLabels map[string]string `json:"extraLabels,omitempty"`
+
+	// CreatePDB sets whether to create a PDB for etcd member Pods.
+	// Note: Disabling this will NOT delete a PDB that was previously created.
+	// Default: true
+	CreatePDB *bool `json:"createPDB,omitempty"`
+
+	// CreateClientService sets whether to create a Service for the client port
+	// of etcd member Pods.
+	// Note: Disabling this will NOT delete a Service that was previously created.
+	// Default: true
+	CreateClientService *bool `json:"createClientService,omitempty"`
+
+	// CreatePeerService sets whether to create a Service for the peer port
+	// of etcd member Pods.
+	// Note: Disabling this will NOT delete a Service that was previously created.
+	// Default: true
+	CreatePeerService *bool `json:"createPeerService,omitempty"`
+
+	// AdvertisePeerURLs can optionally be used to override the URLs that etcd
+	// members use to find each other for peer-to-peer connections.
+	// If specified, the list must contain exactly 3 entries, one for each etcd
+	// member index (1,2,3) respectively.
```

Comment:
do we document that there must be exactly 3 instances somewhere?

Reply:
I try to mention it in every place it's relevant (like here). If you find a place where it's missing, we can add it.